docs/handlers-and-middleware.rst

```diff
@@ -147,7 +147,7 @@ a function that takes the request argument and returns the request to send.
 
     use Psr7\Http\Message\RequestInterface;
 
-    $stack = $client->getHandlerStack();
+    $stack = new \GuzzleHttp\HandlerStack();
```

Comment:
Well, I'm just consistent with some other code examples in this doc.. Just open https://github.com/guzzle/guzzle/blob/master/docs/handlers-and-middleware.rst and find `\GuzzleHttp\HandlerStack`. Even in the same codeblock there is `new \GuzzleHttp\Client` so.. what's the point. We should either make all code examples use imports or stick to FQCN.

Ff we take a closer look in the same codeblock there is `Middleware::mapRequest` which is neither imported nor FQCN.